cli/cmd/deploy/deploy.go

```diff
@@ -55,91 +57,76 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				}
 			}
 
-			// log in if not logged in
-			if !cfg.IsAuthenticated() {
-				warn.Println("In order to deploy to Rill Cloud, you must login.")
-				time.Sleep(2 * time.Second)
-				if err := auth.Login(ctx, cfg); err != nil {
-					return fmt.Errorf("failed to login %w", err)
-				}
-			}
-			adminClient, err := cmdutil.Client(cfg)
-			if err != nil {
-				return err
-			}
-
-			if cfg.Org == "" {
-				// no default org set by user
-				res, err := adminClient.ListOrganizations(context.Background(), &adminv1.ListOrganizationsRequest{})
-				if err != nil {
-					return fmt.Errorf("listing orgs failed with error: %w", err)
-				}
-
-				orgs := res.Organizations
-
-				if len(orgs) == 1 {
-					if err := dotrill.SetDefaultOrg(orgs[0].Name); err != nil {
-						return err
-					}
-				}
-
-				if len(orgs) > 1 {
-					switchCmd := org.SwitchCmd(cfg)
-					switchCmd.SetContext(ctx)
-					if err := switchCmd.RunE(switchCmd, nil); err != nil {
-						return fmt.Errorf("org selection failed %w", err)
-					}
-				}
-			}
-
-			// verify current directory has rill project
+			// Verify that the projectPath contains a Rill project
 			if !hasRillProject(projectPath) {
 				fullpath, err := filepath.Abs(projectPath)
 				if err != nil {
 					return err
 				}
 
-				warn.Printf("\nCurrent path `%s` doesn't have a valid rill project.\n\nPlease run `rill deploy` from correct path or pass correct path via `--project` flag.\n\n", fullpath)
-				warn.Printf("In case there is no valid rill project present, Please use `rill init` to create an empty rill project.\n\n")
+				warn.Printf("\nDirectory at %q doesn't contain a valid Rill project.\n\n", fullpath)
+				warn.Printf(`Please run "rill deploy" from a Rill project directory or use "--project" to pass a project path.\n\n`)
+				warn.Printf(`Run "rill start" to initialize a new Rill project.\n\n`)
```

Comment:
IMO we should tell the users that under what condition is it expected to run `rill start`